lookup.js

```diff
@@ -106,38 +107,39 @@ exports.init = function (sbot, config) {
       details.tombstoned = true
       details.reason = content.reason
     }
-    details.recps = content.recps || null
     return details
   }
 
   function updateLookup(msg) {
     const { type, subfeed, metafeed } = msg.value.content
```

Comment:
style:

```js
const { type, subfeed: id, metafeed: parent } = msg.value.content
```

Reply:
I secretly hate this kind of destructuring because I'm never sure if it's the left side being renamed or the right side, but :shrug: okay